src/main/java/com/money_account_service/dtos/request/CreateAccountRequestDto.java

```diff
@@ -0,0 +1,4 @@
+package com.money_account_service.dtos.request;
+
+public record CreateAccountRequestDto(String currency, String accessToken) {
```

Comment:
I think `accessToken` should not be part of the DTO. Access token should be verified on the controller layer (maybe with some request interceptor). Later, you should operate on the user/principal abstraction.

https://www.baeldung.com/get-user-in-spring-security